cache/backend/sftp_test.go

```diff
@@ -0,0 +1,9 @@
+package backend
+
+import (
+	"testing"
+)
+
+func TestSFTPTruth(t *testing.T) {
+	t.Skip("skipping backend package tests")
```

Comment:
In order to accept the PR we need some tests. You should actually add the test that you have mentioned in  [your comment](https://github.com/meltwater/drone-cache/pull/68#issuecomment-539940243) in here or you can add your tests what we already have in [plugin tests](https://github.com/meltwater/drone-cache/blob/master/plugin/plugin_test.go). 

I have a separate issue to address issues with testing like separating them into files by their concerned backend or functionality. But for now, we can put them together.

You also mentioned an additional docker container that you need for your tests, it's perfectly fine just add that image in our [`docker-compose file`](https://github.com/meltwater/drone-cache/blob/master/docker-compose.yml) and use [test script](https://github.com/meltwater/drone-cache/blob/master/test) to run your tests.